common/api/core-backend.api.md

```diff
@@ -3175,8 +3175,8 @@ export abstract class IModelDb extends IModel {
     get pathName(): LocalFileName;
     performCheckpoint(): void;
     // @internal
-    prepareSqliteStatement(sql: string, logErrors?: boolean): SqliteStatement;
-    prepareStatement(sql: string, logErrors?: boolean): ECSqlStatement;
+    prepareSqliteStatement(sql: string, logErrors?: boolean, persistent?: boolean): SqliteStatement;
+    prepareStatement(sql: string, logErrors?: boolean, persistent?: boolean): ECSqlStatement;
```

Comment:
Please prefer not to add new optional arguments. Consider instead adding an overload that takes a single `args` object.